lib/config/configuration.js

```diff
@@ -745,12 +741,15 @@ Configuration.prototype._loadExcludedFiles = function(masks) {
  * @protected
  */
 Configuration.prototype._loadExtract = function(masks) {
-    if (masks === false) {
+    if (masks === true) {
+        masks = this._defaultExtractFileMasks;
+    } else if (masks === false) {
         masks = [];
+    } else {
+        assert(Array.isArray(masks), '`extract` option should be array of strings');
+        masks = masks.concat(this._defaultExtractFileMasks);
```

Comment:
You still concat default masks. It's not a good idea. Left users to specify exact set of masks.